rust/crd/src/airflowdb.rs

```diff
@@ -40,18 +42,17 @@ type Result<T, E = Error> = std::result::Result<T, E>;
 )]
 #[serde(rename_all = "camelCase")]
 pub struct AirflowDBSpec {
-    pub airflow_version: String,
+    /// The Airflow image version to use
```

Comment:
Nit: By using custom it can be completely other image
```suggestion
    /// The Airflow image to use
```